src/character.h

```diff
@@ -602,7 +605,7 @@ class Character : public Creature, public visitable<Character>
         item weapon;
         item ret_null; // Null item, sometimes returns by weapon() etc
 
-        std::vector<bionic> my_bionics;
+        copyable_unique_ptr<bionic_collection> my_bionics;
```

Comment:
This here is the problem I think, the vector was default contructed, but the unique_ptr is constructed empty, it just needs a new default constructed bionic_collection.

Reply:
all good, added ```my_bionics.reset( new bionic_collection() );``` to the end of the Character constructor.

Reply:
Yes, sorry. I forgot that. Somehow I assumed the wrapper would create the pointer automatically. May need to add something like this.